tests/integration/smb_share_test.go

```diff
@@ -385,6 +385,29 @@ func allSmbShareSuites() map[string]suite.TestingSuite {
 				Password: "1115Rose.",
 			}},
 		}
+		m["smbShareClusteredDMDNS"] = &SmbShareWithDNSSuite{SmbShareSuite{
+			fileSources: []kube.FileSource{
+				{
+					Path:      path.Join(testFilesDir, "joinsecret1.yaml"),
+					Namespace: testNamespace,
+				},
+				{
+					Path:      path.Join(testFilesDir, "smbsecurityconfig2.yaml"),
+					Namespace: testNamespace,
+				},
+				{
+					Path:      path.Join(testFilesDir, "smbshare_ctdb2.yaml"),
+					Namespace: testNamespace,
+				},
+			},
+			smbShareResource: types.NamespacedName{testNamespace, "cshare2"},
+			maxPods:          3,
```

Comment:
It would be good to know what these hard coded numbers mean 3 for maxpods and 4 for minimum number of containers in a ctdb pod.